config/dev-vm-mac.js

```diff
@@ -0,0 +1,16 @@
+const { merge } = require('webpack-merge');
+var base = require('./index.js')
+var devVm  = require('./dev-vm.js')
+
+module.exports = merge(base, devVm, {
+	app: {
+		graphqlUri: 'https://api-vm.kiva.org/fed/graphql',
+	},
+	server: {
+		graphqlUri: 'https://api-vm.kiva.org/fed/graphql',
+		// memcachedEnabled: false,
+		memcachedEnabled: true,
+		memcachedServers: 'localhost:11211',
```

Comment:
This resolves to memcached running in docker. Verify from with `nc localhost 11211`. `stats` to inspect. `quit` to exit.